README.md

```diff
@@ -75,7 +95,7 @@ Relative local dependencies sometimes fail with the following error during Jupyt
 Tarball is not in network and can not be located in cache (["../yarn/v2/.tmp/12c686dd13cb1ef64cebbd243df0a44d/jupyterlab-git.tgz","../yarn/v2/.tmp/064d18f9871b47b33e275a17ec9ead49/.yarn-tarball.tgz"])
 ```
 
-To fix, simply update the local dependency from a relative path to an absolute path.
+To fix, update the local dependency from a relative path to an absolute path.
```

Comment:
Wondering if we can specify instructions to install juptyerlab-git from source. Would that work after I merge in https://github.com/jupyterlab/jupyterlab-git/pull/384/files ?